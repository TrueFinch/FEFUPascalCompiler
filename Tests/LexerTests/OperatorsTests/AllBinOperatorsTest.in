1 + 2 
    1+2
5 - 6
    5-6
3 * 4 
    3*4
7 / 8
    7/8
0 .. 23
    0..23