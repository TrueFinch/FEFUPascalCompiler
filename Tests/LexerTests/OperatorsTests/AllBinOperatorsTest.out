(1, 1)    | IntegerNumber       | 1                             | 1                             |
(1, 3)    | SumOperator         | +                             | +                             |
(1, 5)    | IntegerNumber       | 2                             | 2                             |
(2, 5)    | IntegerNumber       | 1                             | 1                             |
(2, 6)    | SumOperator         | +                             | +                             |
(2, 7)    | IntegerNumber       | 2                             | 2                             |
(3, 1)    | IntegerNumber       | 5                             | 5                             |
(3, 3)    | DifOperator         | -                             | -                             |
(3, 5)    | IntegerNumber       | 6                             | 6                             |
(4, 5)    | IntegerNumber       | 5                             | 5                             |
(4, 6)    | DifOperator         | -                             | -                             |
(4, 7)    | IntegerNumber       | 6                             | 6                             |
(5, 1)    | IntegerNumber       | 3                             | 3                             |
(5, 3)    | MulOperator         | *                             | *                             |
(5, 5)    | IntegerNumber       | 4                             | 4                             |
(6, 5)    | IntegerNumber       | 3                             | 3                             |
(6, 6)    | MulOperator         | *                             | *                             |
(6, 7)    | IntegerNumber       | 4                             | 4                             |
(7, 1)    | IntegerNumber       | 7                             | 7                             |
(7, 3)    | DivOperator         | /                             | /                             |
(7, 5)    | IntegerNumber       | 8                             | 8                             |
(8, 5)    | IntegerNumber       | 7                             | 7                             |
(8, 6)    | DivOperator         | /                             | /                             |
(8, 7)    | IntegerNumber       | 8                             | 8                             |
(9, 1)    | IntegerNumber       | 0                             | 0                             |
(9, 3)    | DoubleDotOperator   | ..                            | ..                            |
(9, 6)    | IntegerNumber       | 23                            | 23                            |
(10, 5)   | IntegerNumber       | 0                             | 0                             |
(10, 6)   | DoubleDotOperator   | ..                            | ..                            |
(10, 8)   | IntegerNumber       | 23                            | 23                            |